extensions/promise/promise.js

```diff
@@ -621,4 +621,26 @@ Using '.thenAll' makes this a bit smoother:\
         
         return promisedType;
     }
+
+    Promise._help_any = {
+        description: "Similar to Promise.all, except it resolves when a single promise provided has succeeded.",
+        returns: "The succeeded promise.",
+        arguments: [
+            {name:"promiseArray", type:"array", description: "The array of promises to evaluate."}
+        ]
+    }
+    Promise.any = function(promiseArray) {

```

Comment:
I would call the parameter "promises" or "promiseIterable", because what's passed in doesn't have to be an array.